data/brands/amenity/bank.json

```diff
@@ -9449,6 +9449,19 @@
         "official_name": "United Services Automobile Association"
       }
     },
+      {
+      "displayName": "Uttarakhand Gramin Bank",
+      "id": "uttarakhandgraminbank-6a1dfb",
+      "locationSet": {"include": ["in"]},
+      "matchNames": ["Uttarakhand Gramin Bank","Gramin Bank"],
```

Comment:
"Uttarakhand Gramin Bank" is exactly same as "brand" and "name", so you dont need to repeat in matchNames. Duplicated matchNames will be removed when npm run build.

Thank you for share those indian brand with us.